### PR TITLE
coffee-unused: minor check contidion

### PR DESCRIPTION
**Pull request by hakankaradis**

### CODE REVIEW

File: index.coffee [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = (folder, skipParseError, callback) ->
   walker.on "file", (root, fileStats, next) ->
     return next() if root in ignoredDirectories
 
-    if fileStats.name.endsWith '.coffee'
+    if fileStats.name.indexOf('.coffee') > -1
```
Comment (sinan) [inline index.coffee]: `if /\.coffee$/.test fileStats.name`

**Comment by cihangir** [comment]
test fails.

**Comment by hakankaradis** [comment]
test passed. @cihangir

**Comment by cihangir** [comment]
👍

**Pull request merged**